vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1640,18 +1640,11 @@ class TimelineFragment @Inject constructor(
         }
     }
 
-    private fun renderSubTitle(typingMessage: String?, topic: String) {
-        // TODO Temporary place to put typing data
-        val subtitle = typingMessage?.takeIf { it.isNotBlank() } ?: topic
+    private fun renderSubTitle(topic: String) {
```

Comment:
I think the aim of #4642 is to completely remove the subtitle, so stop displaying the room topic there.

Reply:
Done.
Thanks